Hyperdrive/Hyperdrive.swift

```diff
@@ -24,32 +24,36 @@ func map<K,V>(source:[K:V], transform:(V -> V)) -> [K:V] {
 }
 
 /// Returns an absolute URI for a URI given a base URL
-func absoluteURI(baseURL:NSURL?)(uri:String) -> String {
-  return NSURL(string: uri, relativeToURL: baseURL)?.absoluteString ?? uri
+func absoluteURI(baseURL: NSURL?) -> (uri: String) -> String {
```

Comment:
These are related to the removal of the currying syntax -> https://github.com/apple/swift-evolution/blob/master/proposals/0002-remove-currying.md